joomla/joomla-cms
# [4.2] Remove useless code

Pull Request for Issue # .

### Summary of Changes
The code which is removed in this PR suppose to index data for every custom fields automatically. However, the code is not working at all. Since PR https://github.com/joomla/joomla-cms/pull/36747 was accepted, we can now choose to index the fields we want and how each field will be indexed, this code should be removed and should speed up the index process (as mentioned, it never worked, so it is useless). See my question https://github.com/joomla/joomla-cms/pull/36747#issuecomment-1133662687 and Hannes's answer https://github.com/joomla/joomla-cms/pull/36747#issuecomment-1133844965

### Testing Instructions
Honestly, I don't know how it should be tested. Maybe you just need to apply patch, try to run index in smart search and make sure it is still working.
